service/frontend/versionChecker.go

```diff
@@ -80,9 +99,20 @@ func (vc *VersionChecker) Stop() {
 	}
 }
 
+func (vc *VersionChecker) RecordSDKInfo(name, version string) {
+	info := sdkNameVersion{name, version}
+	valIface, ok := vc.sdkInfoCounter.Load(info)
+	if !ok {
+		// Store if wasn't added racy
+		valIface, _ = vc.sdkInfoCounter.LoadOrStore(info, &sdkCount{})
+	}
+	atomic.AddInt64(&valIface.(*sdkCount).count, 1)
```

Comment:
the counter will essentially be request count make by this type of sdk?

Reply:
I think we have decided not to collect any stats.

Reply:
yes, it is reset before we send the request

Reply:
I can change this to be a set instead of counting